config/locales/find.yml

```diff
@@ -28,8 +28,8 @@ en:
         salary: Salary
         apprenticeship: Salary (apprenticeship)
         fee:
-          uk_fees_html: "%{value} for UK citizens"
-          international_fees_html: "%{value} for Non-UK citizens"
+          uk_fees_html: "Fee - UK citizens %{value} "
+          international_fees_html: "Fee - Non-UK citizens %{value}"
```

Comment:
This is the outdated solution 😅 It should be "£4,500 fee for UK citizens" as mentioned in the latest comment, [here](https://trello.com/c/Jq22IvMJ#comment-67c6c69fe0f23d91516fed34).